src/actions/placeBid.ts

```diff
@@ -159,6 +141,9 @@ export const placeBid = async ({
     wallet,
     txs: txBatch.toTransactions(),
     signers: txBatch.signers,
+    options: {
+      ...(commitment && { commitment }),
+    },
```

Comment:
Is this different from just `options: commitment ?? {}` which is arguably easier to read?